rest-api/test/unit_test/dao_test/participant_summary_dao_test.py

```diff
@@ -254,10 +256,13 @@ def add_sample(participant, test_code, sample_id):
     M_baseline_samples = self._insert(Participant(participantId=9, biobankId=99))
     add_sample(M_baseline_samples, baseline_tests[0], '999')
     M_first_update = self.dao.get(M_baseline_samples.participantId)
-
+    # sleep 1 sec to make lastModified different
+    time.sleep(1)
     self.dao.update_from_biobank_stored_samples()
```

Comment:
the precision of MySQL only to second, so we need to sleep at least 1 sec to make the lastModify timestamp change.